cimr/processor/utils.py

```diff
@@ -12,6 +12,25 @@
 import logging
 import subprocess
 
+def set_chrom_dict():
+    """Make a dictionary to standardize chromosome IDs in input files."""
+    maxchrom = 23
+    chrom_dict = {str(i):'chr' + str(i) for i in range(1, maxchrom)}
+    chrom_dict.update({'chr'+str(i):'chr' + str(i) for i in range(1, maxchrom)})
+    chrom_dict.update({
+        'X':'chr23', 
+        'Y':'chr24', 
+        'M':'chr25', 
+        'MT':'chr25', 
+        'chrX':'chr23', 
+        'chrY':'chr24', 
+        'chrM':'chr25', 
+        'chrMT':'chr25'
+    })
+    chrom_str = ['chr' + str(i) for i in range(1, maxchrom)]
+    chrom_int = [i for i in range(1, maxchrom)]
+    return chrom_dict, maxchrom, chrom_str, chrom_int
```

Comment:
So `chrom_str` and `chrom_int` don't need X/Y/M/MT/23/24/25?

Reply:
Yeah.. For future analyses purposes any inputted data will be checked and saved but the current list of tools to be included in the cimr release will not use non-autosomal chromosomes.